vignettes/01_rphenoscape.Rmd

```diff
@@ -51,12 +51,12 @@ The [RPhenoscape] package provides convenient access to the Phenoscape Knowledge
 library(rphenoscape)
 ```
 
-First let’s look up information about the query terms. You can use the term detail functions ```pk_anatomical_detail()```, ```pk_taxon_detail()```, ```pk_phenotype_detail()```, and ```pk_gene_detail()``` to retrieve an ontology term’s ID, label, and definition. 
+First let’s look up information about the query terms. You can use the term detail functions ```anatomy_term_info()```, ```taxon_info()```, ```phenotypic_quality_term_info()```, and ```gene_info()``` to retrieve an ontology term’s ID, label, and definition. 
 
 For example, view the details for the anatomy term “fin”:
 
 ```{r}
-pk_anatomical_detail("fin")
+anatomy_term_info("fin")
```

Comment:
See examples at the [as.terminfo() reference](https://rphenoscape.phenoscape.org/reference/terminfo.html) for the recommended (and much more transparent) way to do this.